src/components/LinkCard.js

```diff
@@ -0,0 +1,83 @@
+import '../styles/LinkCard.css';
+import star from '../assets/star.svg';
+import kebab from '../assets/kebab.svg';
+
+const getElapsedTimeMessage = createdAtTime => {
+  const currentDays = new Date();
+  let ElapsedTime = Math.floor((currentDays - createdAtTime) / 1000 / 60);
+
+  if (ElapsedTime < 2) {
+    return '1 minute ago';
+  }
+
+  if (ElapsedTime < 60) {
+    return `${ElapsedTime} minutes ago`;
+  }
+
+  ElapsedTime = Math.floor(ElapsedTime / 60);
+
+  if (ElapsedTime < 24) {
+    return `${ElapsedTime} ${ElapsedTime === 1 ? 'hour' : 'hours'} ago`;
+  }
+
+  ElapsedTime = Math.floor(ElapsedTime / 24);
+
+  if (ElapsedTime < 31) {
+    return `${ElapsedTime} ${ElapsedTime === 1 ? 'day' : 'days'} ago`;
+  }
+
+  ElapsedTime = Math.floor(ElapsedTime / 365);
+
+  if (ElapsedTime < 1) {
+    const currentYear = currentDays.getFullYear();
+    const currentMonth = currentDays.getMonth() + 1;
+    const currentDate = currentDays.getDate();
+    const createdYear = createdAtTime.getFullYear();
+    const createdMonth = createdAtTime.getMonth() + 1;
+    const createdDate = createdAtTime.getDate();
+
+    if (currentYear === createdYear) {
+      if (currentDate >= createdDate) {
+        return `${currentMonth - createdMonth} months ago`;
+      }
+      return `${currentMonth - createdMonth - 1} months ago`;
+    }
+    if (currentYear > createdYear) {
+      if (currentDate >= createdDate) {
+        return `${12 - createdMonth + currentMonth} months ago`;
+      }
+      return `${12 - createdMonth + currentMonth - 1} months ago`;
+    }
+  }
+
+  if (ElapsedTime >= 1) {
+    return `${ElapsedTime} ${ElapsedTime === 1 ? 'year' : 'years'} ago`;
+  }
+};
+
+function LinkCard({ url, createdAt, desc, imgUrl }) {
+  const createdDays = new Date(createdAt);
+
+  return (
+    <a href={url} target="_blank" rel="noopener noreferrer" className="link-card-area">
+      <div className="img-group">
+        {imgUrl ? <img src={imgUrl} alt="미리보기" className="img-card" /> : <div className="img-card no-img"></div>}
+        <button className="btn-star">
+          <img src={star} alt="별모양" />
+        </button>
+      </div>
+      <div className="card-info-group">
+        <div className="time-info-group">
+          <p className="elapsed-time">{getElapsedTimeMessage(createdDays)}</p>
+          <button className="more">
+            <img src={kebab} alt="더보기" />
+          </button>
+        </div>
+        <p className="desc">{desc}</p>
+        <p className="date">{`${createdDays.getFullYear()}. ${createdDays.getMonth() + 1}. ${createdDays.getDate()}`}</p>

```

Comment:
jsx 내부에서만 알 수 있는 값이 아닌 경우 그냥 위에서 상수화해주시고 여기에는 결과만 적어주시는 게 연산 횟수를 줄일 수 있어요!
```suggestion
const dateInfo = `${createdDays.getFullYear()}. ${createdDays.getMonth() + 1}. ${createdDays.getDate()}`
        <p className="date">{dateInfo}</p>

```

Reply:
jsx 내부에서만 알 수 있는 값이라는 말이 이해가 잘 안됩니다